core/net/ipv6/uip-nd6.c

```diff
@@ -518,7 +518,7 @@ na_input(void)
         goto discard;
       }
 
-#if NETSTACK_RADIO != ble_mode
+#if NETSTACK_RADIO != ble_mode_driver
```

Comment:
That's an interesting pull request!

I don't think C preprocessor can see symbolic names, so I suspect this check cannot be done this way. For an example, try compiling this code - it will print "equals":
```
int abc, def;
#define ABC abc
#if ABC == def
#warning equals
#endif
```